index.js

```diff
@@ -131,6 +131,11 @@ function exec(str, cols, opts) {
 			rows.push('');
 		}
 
+		if (options.wordWrap === false) {
```

Comment:
I think I can optimise this by only executing `wrapWord` when the total length would exceed the number of columns. Otherwise we can just append.